src/sparql/qc/general/qc-duplicate-exact-synonym-no-abbrev.sparql

```diff
@@ -31,7 +31,7 @@ SELECT DISTINCT ?entity ?property ?value WHERE {
          oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/mondo#ABBREVIATION> .
   }
   
-  FILTER NOT EXISTS { ?entity owl:deprecated true }
+  FILTER NOT EXISTS { ?entity1 owl:deprecated true }
```

Comment:
## Super minor code style thing
I suppose in the this SPARQL, we could add a space before the first `.` below for consistency.
```
  ?entity1 ?property1 ?value.
  ?entity2 ?property2 ?value .
```

Reply:
agreed, although i've seen this other queries and will leave as-is for now.